package.json

```diff
@@ -1,50 +1,45 @@
 {
   "name": "react-page-visibility",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "description": "Declarative, nested, stateful, isomorphic page visibility for React",
   "author": "Gilad Peleg <giladp007@gmail.com> (http://giladpeleg.com)",
   "repository": "pgilad/react-page-visibility",
-  "keywords": "react, components, visibility, page, higher-order, browser",
-  "main": "index.js",
+  "keywords": "react, components, visibility, page, higher-order, browser, react-component",
+  "main": "dist/index.js",
+  "module": "src/index.js",
```

Comment:
Just to note, remember to update the `.babelrc` config so that you export `{ modules: false }` for the pure build (non umd) as otherwise babel will wrap everything in commonjs, meaning anyone using webpack won't be able to get the benefits of tree shaking 😭